src/_data/companies.yml

```diff
@@ -2059,3 +2059,15 @@ Bizneo HR:
   url: https://www.bizneo.com/
   location: Madrid, Spain
   description: "Bizneo’s HR Suite offers tools to manage your entire employee lifecycle. Built with Elixir and Phoenix, our HR management tools empower companies to act quickly and think globally in their HR processes."
+
+Ardhika Softwares:
+  industry: Computer Software/Engineering
```

Comment:
One last thing @gopithegeek, it looks like the alignment for something here is off.  `industry` and `url` are not aligned.

Reply:
@doomspork thanks for bringing it to notice 👍 
have edited